### PR TITLE
removed deprecation errors from php-cs-fixer fix

### PR DESCRIPTION
**Pull request by xBazilio**
This fix should help another my PR to pass all checks

### CODE REVIEW

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -65,7 +65,7 @@ public function supports(\SplFileInfo $file) : bool
 file that was distributed with this source code.
 EOF;
 
-return Config::create()
+return (new Config)
```
Comment (rybakit) [inline .php-cs-fixer.dist.php]:
```suggestion
return (new Config())
```

**Comment by rybakit** [comment]
@xBazilio Could you also bump the php-cs-fixer dependency in composer.json to `^2.19` (since this is the version where the new file name was introduced), and update `.gitignore` and `.gitattrubutes`?

File: .gitignore [hunk 2]
```diff
@@ -3,3 +3,4 @@
 /composer.lock
 /coverage*
 /phpunit.xml
+/.idea
```
Comment (rybakit) [inline .gitignore]: Sorry for not being clear enough, I meant adding 
```suggestion
/.php-cs-fixer.php
```

Comment (xBazilio) [reply .gitignore]: I didn't see the old file there so I didn't get it.

**Comment by rybakit** [comment]
Merged, thanks!

**Pull request merged**